api/api_storage.go

```diff
@@ -48,6 +48,11 @@ type StorageMiner interface {
 	ActorSectorSize(context.Context, address.Address) (abi.SectorSize, error) //perm:read
 	ActorAddressConfig(ctx context.Context) (AddressConfig, error)            //perm:read
 
+	// WithdrawBalance allows to withdraw balance from miner actor to owner address
+	// Specify amount as "0" to withdraw full balance. This method returns a message CID
+	// and does not wait for message execution
+	WithdrawBalance(ctx context.Context, amount abi.TokenAmount) (cid.Cid, error) //perm:admin
```

Comment:
One final nitpick - can we call this method `ActorWithdrawBalance`, to keep it grouped with the other methods which are related to the on-chain actor?

Reply:
Sure. I will rename it to ActorWithdrawBalance

Reply:
@magik6k Done